app/assets/javascripts/pages/agent-show-page.js.coffee

```diff
@@ -40,6 +41,12 @@ class @AgentShowPage
         $("#logs .spinner").stop(true, true).fadeOut ->
           $("#logs .refresh, #logs .clear").show()
 
+  toggleMemory: ->
+    if $('#memorypanel').hasClass('hidden')
+      $('#memorypanel').removeClass 'hidden'
```

Comment:
It looks like the button text does not change anymore when you show the memory, is that intended?